packages/fscomponents/src/components/MultiCarousel/MultiCarousel.tsx

```diff
@@ -84,24 +84,27 @@ export class MultiCarousel<ItemT> extends Component<MultiCarouselProps<ItemT>, M
     this.state = {
       currentIndex: 0,
       containerWidth: 0,
-      itemWidth: 0
+      itemWidth: 0,
+      opacity: new Animated.Value(0),
+      initialized: false
```

Comment:
This is probably the wrong way to handle the animation. We should instead use [`componentDidUpdate `](https://reactjs.org/docs/react-component.html#componentdidupdate) to set the opacity to zero and animate in whenever the image changes. We shouldn't need to keep track of `initialized` at all.